pkg/util/mount/mount_linux.go

```diff
@@ -1003,10 +1003,11 @@ func (mounter *Mounter) SafeMakeDir(subdir string, base string, perm os.FileMode
 }
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
-	if _, err := os.Stat(pathname); os.IsNotExist(err) {
+	pathExists, pathErr := PathExists(pathname)
+	if !pathExists || IsCorruptedMnt(pathErr) {
 		return []string{}, nil
-	} else if err != nil {
-		return nil, err
+	} else if pathErr != nil {
+		return nil, fmt.Errorf("Error checking path: %v", pathErr)
```

Comment:
do we need to print out the path or the pathErr should already include it?

Reply:
resolved